addons/mail/static/src/core/common/message_model.js

```diff
@@ -74,6 +74,14 @@ export class Message extends Record {
             return this.id < this.thread.lastSelfMessageSeenByEveryone.id;
         },
     });
+    isReadBySelf = Record.attr(false, {
+        compute() {
+            return (
+                this.thread.selfMember?.seen_message_id?.eq(this) &&
```

Comment:
Shouldn't the seen_message_id also compared with `>=` rather than `=`?

I think it works because we only ever call this on the last message of thee thread, but if we used this field on a previous than last (for example last being deleted?) then it would not work?

Reply:
Yes it should, updated.